### PR TITLE
Changes in Tableu De Board report

### PR DESCRIPTION
**Pull request by kkrampa**

### CODE REVIEW

**Comment by kkrampa** [comment]
@czue @snopoke 
Mostly tests and small refactor.

File: custom/intrahealth/sqldata.py [hunk 1]
```diff
@@ -183,58 +184,46 @@ def group_by(self):
...
+        products = self.products
+        values = {
+            product.product_id: [0, 0, 0]
+            for product in products
+        }
+
+        rows = super(DispDesProducts, self).rows
+        for row in rows:
+            product_id = row[0]
+            values[product_id] = [row[1], row[2], "%d%%" % (100 * row[2]['html'] / (row[1]['html'] or 1))]
```
Comment (snopoke) [inline custom/intrahealth/sqldata.py]: Might make sense to override the `rows` property to use a `DictDataFormat` which would make this much easier to read. I think you can just call `self.get_data()` to get a list of dicts as the rows

**Comment by kkrampa** [comment]
One more fix and test. 
@snopoke I changed rows according to your suggestion. Thanks!

**Comment by czue** [comment]
one of the test fails still related
```
======================================================================
FAIL: test_disp_des_products_report (custom.intrahealth.tests.test_reports.TestReports)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/custom/intrahealth/tests/test_reports.py", line 58, in test_disp_des_products_report
    ['Taux', '88%', '100%']
AssertionError: Lists differ: [['Commandes', {'sort_key': 25... != [['Commandes', {'sort_key': 26...
First differing element 0:
['Commandes', {'sort_key': 25L, 'html': 25L}, {'sort_key': 26L, 'html': 26L}]
['Commandes', {'sort_key': 26L, 'html': 26L}, {'sort_key': 25L, 'html': 25L}]
Diff is 680 characters long. Set self.maxDiff to None to see it.
```

**Comment by kkrampa** [comment]
@czue Could you kick off build again?

**Comment by czue** [comment]
:boot:

**Comment by kkrampa** [comment]
:white_check_mark:

**Pull request merged**